### DISCUSSION TITLE
Per project Pro license for an app?

### DISCUSSION THREAD

**Discussion by AsyncPiMaker** [statamic/cms#4369] [Q&A]
Hi,

If I am evaluating making an app based on Statamic for SAAS server management, is that one project?

In other words, if I take Statamic Pro and build a standard admin interface to a separate application, if I deploy it on AWS as an AMI, is that one project?  

Am I missing something? Each deployment will be a different URL, with Statamic behind a proxy server.

So, if I have 100 of these, is that a 100 licenses or 1 license for Pro?

Thanks

**Comment by dgedwards13** [comment]
My understanding is that would have to be 100 licenses. 

From the docs:

> Each license entitles you to run one production installation. You will need to specify the domains you plan to use from the “Sites” area of your Statamic Account.

> If you attempt to use the site from a domain not listed in your Site settings, you will get a notification inside the Control Panel informing you thusly to make the necessary changes.

> Unless you have been granted prior, written consent from Statamic, you may not:
> * Reproduce, distribute, or transfer the Software, or portions thereof, to any third party.
> * Sell, rent, lease, assign, or sublet the Software or portions thereof.
> * Grant rights to any other person.

For more details, check out the license fine print found [here](https://statamic.com/license): https://statamic.com/license

**Comment by duncanmcclean** [comment]
Will each site have it's own CP - then yes. 100 Pro licenses.

Also, be aware you can't resell Statamic under another name, it's against the license agreement.

**Comment by 1986webdeveloper** [comment]
I did this one of our product already. and @jackmcdade will soon be ready with APIs, it should be per site license
